qmc2.pro

```diff
@@ -335,6 +335,7 @@ contains(DEFINES, QMC2_BUNDLED_MINIZIP) {
 
 contains(DEFINES, QMC2_BUNDLED_ZLIB) {
 	INCLUDEPATH += src/zlib
+	QMAKE_CFLAGS += -DHAVE_UNISTD_H
```

Comment:
Does this have potential to cause issues on targets that aren’t vaguely UNIX-like (e.g. Windows)?  For MAME, which may have a newer version of zlib source bundled, the only preprocessor macros we define in the build scripts are `verbose` and `ZLIB_CONST`?